src/components/content-types/TextContent.tsx

```diff
@@ -31,8 +31,15 @@ import {
   useLastSeenHeads,
 } from "../pushpin-code/Changes"
 import { createDocumentLink } from "../pushpin-code/Url"
-import memoize from "lodash.memoize"
-import { Doc, getHeads } from "@automerge/automerge"
+import { Doc, getHeads, Heads, view } from "@automerge/automerge"
+import {
+  evalAllSearches,
+  evalSearchFor,
+  Match,
+  MENTION,
+} from "../pushpin-code/Searches"
```

Comment:
this is an aside but we should rename pushpin-code to a more intellgible name. that was just where i put code that i pulled in from the pushpin project file-by-file as i discovered the minimum required set of stuff.

Reply:
I agree we should have a general cleanup patch after all the pull requests are merged. There are also a lot of unused imports and things like that